src/components/disclaimer.tsx

```diff
@@ -0,0 +1,23 @@
+import Config from "../config";
+
+export default function DisclaimerBanner() {
+  if (Config.production()) {
+    return (
+      <div className="flex justify-center w-full bg-yellow-400 p-4">
```

Comment:
It's good but slightly repetitive. The same style and layout is built twice when it could just be done once; instead, just the text/inner content could be changed depending on the case.